src/core/lib/gprpp/status_helper.cc

```diff
@@ -230,7 +230,10 @@ absl::optional<absl::Time> StatusGetTime(const absl::Status& status,
   if (p.has_value()) {
     absl::optional<absl::string_view> sv = p->TryFlat();
     if (sv.has_value()) {
-      return *reinterpret_cast<const absl::Time*>(sv->data());
+      // copy the content before casting to avoid misaligned address access
+      char buf[sizeof(const absl::Time)];
```

Comment:
How does copying to a char array avoid misaligned address access?
don't we need ```alignas(absl::Time) char buf[sizeof(const absl::Time)]```?

Reply:
Good catch. Done!